.github/workflows/add-to-project.yaml

```diff
@@ -0,0 +1,26 @@
+name: Add issue to project
+
+on:
+  workflow_call:
+    inputs:
+      project_id:
+        type: number
+        required: true
+jobs:
+  add-issue-to-project:
+    name: Add issue with `status/triage` label to project
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'status/triage' }}
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@v1.2.1
```

Comment:
Please lock to commit sha, renovate can keep track of the version tag comments and update it via PRs.

Why to pin you may ask? 🙂 
https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash (+ many more other resources out there)

```suggestion
        uses: actions/create-github-app-token@49ce228ea7cddec9f88dd09c5b7740dbac82d7ba 
# v
```